ios/SecuritySdk/Classes/Digital Signature/Secure Enclave/SecureEnclaveDigitalSignatureKeyManager.swift

```diff
@@ -0,0 +1,84 @@
+//
+//  SecureEnclaveDigitalSignatureKeyManager.swift
+//  SecuritySdk
+//
+
+import Foundation
+
+public class SecureEnclaveDigitalSignatureKeyManager: DigitalSignatureKeyManager {
+    
+    // MARK: - Internal Properties
+    
+    internal typealias PrivateKey = SecureEnclaveSigningKey
+    internal typealias PublicKey = SecureEnclaveVerifyingKey
+    
+    // MARK: - Private Properties
+    
+    private let tag: String
+    
+    // MARK: - Initialization
+    
+    public init(tag: String) {
+        self.tag = tag
+    }
+
+    // MARK: - Public Methods (DigitalSignatureKeyManager)
+    
+    public func signingKey() throws -> SecureEnclaveSigningKey {
+        let checkIfExists = try keyExists()
+
+        if checkIfExists {
+            var keyRef: CFTypeRef?
+            let preparedQuery = SecureEnclaveKeychainQueries.getQuery(with: tag, returnRef: true)
+            SecItemCopyMatching(preparedQuery, &keyRef)
+            return try SecureEnclaveSigningKey(secKey: keyRef as! SecKey)
+        }
+
+        return try SecureEnclaveSigningKey(secKey: generateKeypair())
```

Comment:
Style suggestion: You could streamline this a bit by using a `guard` here to move the generate path up, which will let you keep the happy path un-indented:

```suggestion
        guard try keyExists() else {
            return try SecureEnclaveSigningKey(secKey: generateKeypair())
        }

        var keyRef: CFTypeRef?
        let preparedQuery = SecureEnclaveKeychainQueries.getQuery(with: tag, returnRef: true)
        SecItemCopyMatching(preparedQuery, &keyRef)
        return try SecureEnclaveSigningKey(secKey: keyRef as! SecKey)
```